Documentation/ApiOverview/Xclasses/Index.rst

```diff
@@ -14,17 +20,19 @@ Introduction
 
 XCLASSing is a mechanism in TYPO3 CMS to extend classes or overwrite methods from the Core or extensions
 with one's own code. This enables a developer to easily change a given functionality,
-if other options like :ref:`hooks <hooks>`, signals or the Extbase dependency injection mechanisms
-do not work or do not exist.
+if other options like :ref:`hooks <hooks>`, signals, :ref:`events <EventDispatcher>`
+or the Extbase dependency injection mechanisms do not work or do not exist.
```

Comment:
I would remove "Extbase" here as we now have dependency injection everywhere in the core.